geomesa-tools/conf/geomesa-env.sh

```diff
@@ -28,6 +28,7 @@ function setvar() {
     export $1=$2
   fi
 }
+GEOMESA_TOOLS_VERSION=%%gmtools.version%%
```

Comment:
@aheyne i have a better way that we can do to use a filtered source file to insert it.